docs/api/hooks/useAsyncError.md

```diff
@@ -10,7 +10,7 @@ title: useAsyncError
 
 [Reference Documentation ↗](https://api.reactrouter.com/v7/functions/react_router.useAsyncError.html)
 
-Returns the rejection value from the closest [Await](../components/Await).
+Returns the rejection value from the closest [`<Await>`](../components/Await).
```

Comment:
This MD link was auto-expanded from `{@link Await | <Await>}` in the source.  The code uses a lookup table of all files in `docs/api` to determine the right path.